README.md

```diff
@@ -3,12 +3,20 @@
 
 Terraform module for deploying Materialize on AWS Cloud Platform with all required infrastructure components.
 
-> **Warning** This is provided on a best-effort basis and Materialize cannot offer support for this module.
-
 The module has been tested with:
+
 - PostgreSQL 15
 - Materialize Helm Operator Terraform Module v0.1.8
 
+> [!WARNING]
+> This module is provided on a best-effort basis and Materialize cannot offer support for it.
```

Comment:
I might nix this, I think we would support people trying to use it to a degree, like if the TF provider clearly had bugs